raven/base.py

```diff
@@ -636,7 +637,10 @@ def capture(self, event_type, data=None, date=None, time_spent=None,
             **kwargs)
 
         # should this event be sampled?
-        if self._random.random() < self.sample_rate:
+        if sample_rate is None:
```

Comment:
thoughts on type checking here? There's no check Client.__init__, so deferring this to the consumer might be fine(?) on the other hand, being defensive isn't the worst idea 🤔